code/products/mysql/connect-mysqlx.py

```diff
@@ -0,0 +1,27 @@
+import mysqlx
+
+MYSQLX_PASSWORD = "user-password"
+SERVICE_URI = "aiven-service-URI"
+
+connection_data = (
+    f"mysqlx://avnadmin:{MYSQLX_PASSWORD}@{SERVICE_URI}/defaultdb?ssl-mode=REQUIRED"
```

Comment:
Does SERVICE_URI include port here? I put another comment below, but I think it is worth adding MYSQLX_PORT separately as a variable to avoid double meaning and confusion.

Reply:
The SERVICE_URI for MySQL service already includes the port. I believe it would be confusing for the user if referred with the port. Because in the UI, you get already in this format. For example:

`mysql://avnadmin:whatever@mysql-name-dev-project.aivencloud.com:PORT/defaultdb?ssl-mode=REQUIRED`